### PR TITLE
Tabs component - fixed width

### LINKED ISSUE #195: Tabs component - Fixed Width

### PR DESCRIPTION
**Pull request by keithluchtel**
Adds implementation for Fixed-Width Tabs from UI Library.

Resolve #195

### CODE REVIEW

File: apps/box/src/screens/Settings/ComponentGallery/TabsDemo.screen.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  FxText,
+} from '@functionland/component-library';
+import React from 'react';
+
+const TAB_ITEMS = ['first', 'second'];
+
+export const TabsDemoScreen = () => {
+  const [selectedIdx, setSelectedIdx] = React.useState<number>(0);
+
+  const selectHander = (idx: number) => {
```
Comment (JDMathew) [inline apps/box/src/screens/Settings/ComponentGallery/TabsDemo.screen.tsx:15]: *Handler

File: libs/component-library/src/lib/tabs/tabs.tsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+export const FxTabs = ({
+  items,
+  selectedIdx = 0,
+  onSelect,
+  animate = true,
+  animationDuration = ANIMATION_DURATION,
+}: FxTabsProps) => {
+  const translateX = useSharedValue(0);
+  const containerWidth = useSharedValue<number>(0);
+  const hightlightWidth = useDerivedValue(() => {
```
Comment (JDMathew) [inline libs/component-library/src/lib/tabs/tabs.tsx:80]: *highlightWidth

File: libs/component-library/src/lib/tabs/tabs.tsx [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+        args.pressed && {
+          opacity: 1,
+          backgroundColor: theme.colors.backgroundPrimary,
+          borderBottomColor: theme.colors.backgroundSecondary,
+        },
+      ]}
+    >
+      <FxText variant="bodySmallRegular" color="content3" {...textOverrides}>
+        {text}
+      </FxText>
```
Comment (JDMathew) [inline libs/component-library/src/lib/tabs/tabs.tsx:59]: As discussed in huddle, it might be good for the text to animate too.

Comment (keithluchtel) [reply libs/component-library/src/lib/tabs/tabs.tsx:59]: I'm going to look more at this in a separate PR. I played around with it for a bit and was having some difficulties getting it where I wanted it.

**Review by JDMathew — APPROVED** [review]

**Pull request merged**